test_util/aws.py

```diff
@@ -99,6 +97,18 @@ def create_stack(self, name, template_url, parameters, deploy_timeout=60):
             # so that we may pass parameters directly from yaml (which parses numbers as non-strings)
             Parameters=param_dict_to_aws_format(parameters))
 
+    def create_stack_from_string(self, name, template, parameters, deploy_timeout=60):
```

Comment:
Ah, so they're generating the library from schema definitions. That's slick. Thanks for the explanation. 👍